SmartServoGui/src/widgetSerialScan.cpp

```diff
@@ -163,34 +181,31 @@ int widgetSerialScan::getCurrentDeviceClass(int index)
 
     switch (index)
     {
-    case 1:
-    case 11:
-        if (ui->comboBox_protocol->currentIndex() == 0)
+    case Manual:
+    case Saved:
+        if (ui->comboBox_protocol->currentProtocol() == ServoProtocol::PROTOCOL_DXLv1)
             deviceClass = SERVO_MX;
-        else if (ui->comboBox_protocol->currentIndex() == 1)
+        else if (ui->comboBox_protocol->currentProtocol() == ServoProtocol::PROTOCOL_DXLv1)
```

Comment:
should be v2! took me a while to figure that out :p

Reply:
Sorry about that, fixing it now.